lib/live_debugger_refactor/api/system/dbg.ex

```diff
@@ -0,0 +1,98 @@
+defmodule LiveDebugger.API.System.Dbg do
```

Comment:
Same as in case of Hubert
https://github.com/software-mansion/live-debugger/pull/565/files#r2191565477